Jenkinsfile

```diff
@@ -4,7 +4,7 @@ def build(arch, mode) {
     // FIXME Technically we could build on linux as well!
     node('osx && git && android-ndk') {
       unstash 'sources'
-      sh "./build_v8.sh -n /opt/android-ndk-r11c -j8 -l ${arch} -m ${mode}"
+      sh "./build_v8.sh -n /opt/android-ndk-r14b -j8 -l ${arch} -m ${mode}"
```

Comment:
and up here, don't point to /opt/android-ndk-r14b because no build machine has that path yet. Instead have it use the android sdk you downloaded earlier in the initial phases and states in 'sources' and unstashed here. so:

sh "./build_v8.sh -n ./android-ndk-r14b -j8 -l ${arch} -m ${mode}"